docs/data/external-oasis-repos.md

```diff
@@ -114,18 +124,17 @@ Procedure to add an externally-hosted repository to OASIS
         :::console
         root@host # echo "5 4 \* \* \* ownerid cd /srv/cvmfs/repo.opensciencgrid.org && wget -qO .cvmfswhitelist.new http://oasis.opensciencegrid.org/cvmfs/repo.opensciencgrid.org/.cvmfswhitelist && mv .cvmfswhitelist.new .cvmfswhitelist" >>/etc/cron.d/fetch-cvmfs-whitelist
 
-    Note that this eliminates the need for the repository service administrator to periodically use "cvmfs\_server resign" to update .cvmfswhitelist. Then the repository service administrator goes back to the open GOC ticket and asks to proceed to step \#7.
+    Note that this eliminates the need for the repository service administrator to periodically use "cvmfs\_server resign" to update .cvmfswhitelist. Then the repository service administrator goes back to the open GOC ticket and asks to proceed to step \#9.
```

Comment:
Switch to preformatted text - `cvmfs_server resign` instead of "cvmfs\_server resign".  Same for  `.cvmfswhitelist`

Reply:
Done